packaging/nnstreamer.spec

```diff
@@ -165,6 +165,7 @@ BuildRequires:  pkgconfig(libpng)
 %if 0%{?flatbuf_support}
 # for flatbuffers
 BuildRequires: flatbuffers-devel
+BuildRequires: flatbuffers-python
```

Comment:
If it's mandatory for `nnstreamer-flatbuf`, you may need `Requires: flatbuffers-python` in line 378.
I don't think it can be automatically resolved because it's not a devel package.

Reply:
What if we add `Requires: flatbuffers-python`, then the target platform should have a python environment. `flatbuffers` is used for a light-weight device such as IoT but python is not. Becuase of this reason, `Requires` might make a image break issue.

What about adding a use case to README.md for advanced platform developers?

Reply:
Then, how about `Recommends` or `Suggests`.

Reply:
Well, it looks good.

Reply:
Thanks! `flatbuffers-python` is added as recommends in flatbuf package.
And the package is installed when `unit_test` is enabled.